agent/src/containerutilityinterface.h

```diff
@@ -61,15 +65,24 @@ class ContainerUtilityInterfaceError : public SoftwareContainerError
  * This class contains utility functions
  */
 
-class ContainerUtilityInterface {
+class ContainerUtilityInterface :
+    private FileToolkitWithUndo {
```

Comment:
Put the brace on the next line or put the inheritance stuff on the same line please.